sites/public/src/components/listing/ListingView.tsx

```diff
@@ -626,7 +626,7 @@ export const ListingView = (props: ListingProps) => {
           {!applicationsClosed && getWaitlist()}
           {hasNonReferralMethods &&
           !applicationsClosed &&
-          listing.status !== ListingsStatusEnum.closed ? (
+          (listing.status !== ListingsStatusEnum.closed || props.preview) ? (
```

Comment:
Won't this still have the issue of if they are trying to reopen a listing that the application due date is in the past? I guess they shouldn't be publishing a listing like that, but they might want to still preview it. And even in that case an easy fix for them would to update the application due date to something in the future.

Reply:
It will still have that behavior but my thought was that it wouldn't be an issue because it is representing what the listing would show when published. I know some admin use preview as a final "does everything look good" check and I'd prefer a closed application flagged to them then.